pytket/phir/cli.py

```diff
@@ -46,13 +46,6 @@ def main() -> None:
         default="H1-1",
         help="Machine name, H1-1 by default",
     )
-    parser.add_argument(
-        "-o",
-        "--tket-opt-level",
-        choices=["0", "1", "2"],
-        default="0",
-        help="TKET optimization level, 0 by default",
-    )
```

Comment:
As this argument is removed, could you update the readme with the output of `phirc -h`? You may need to issue `make dev-all` before being able to use the CLI.